aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md

```diff
@@ -46,9 +46,9 @@ VSTS provides a number of features to assist you with application development fr
 - It supports agile project management.
 
 
-For more information on the continuous integration and delivery features of VSTS, see [Visual Studio Team Services](https://www.visualstudio.com/team-services/).
+For more information on the continuous integration and delivery features of Azure DevOps Services, see [the Azure DevOps documentation](/azure/devops/index).
 
-If you're looking for a turn-key project management, team collaboration, and source control solution, check out VSTS. The service is free for up to 5 users, and you can sign up for it at [Visual Studio Team Services](https://www.visualstudio.com/team-services/).
+If you're looking for a turn-key project management, team collaboration, and source control solution, check out Azure DevOps Services. The service is free for up to 5 users, and you can sign up for it at [Azure DevOps Services](https://dev.azure.com/).
```

Comment:
The style manual prefers that docs don't say if something is "free" or specify a particular offer. IIRC, they imply or state that it can create a legal problem for MS if the offer changes but the doc doesn't.

Reply:
Can you change the last sentence to just "You can sign up for it at [Azure DevOps Services](https://dev.azure.com/)." ?

Reply:
Or should I update the PR?

Reply:
Sure ... I'll take care of it.

Reply:
I made it a straight call for action ...

> Sign up at \[Azure DevOps Services](https://dev.azure.com/).